app/Console/Commands/InitCommand.php

```diff
@@ -154,22 +169,35 @@ private function maybeSetMediaPath(): void
             return;
         }
 
-        $this->info('The absolute path to your media directory. If this is skipped (left blank) now, you can set it later via the web interface.');
-
-        while (true) {
-            $path = $this->ask('Media path', false);
-
-            if ($path === false) {
-                return;
-            }
+        if ($this->inNoInteractionMode()) {
+            $path = config('koel.media_path');
 
             if (is_dir($path) && is_readable($path)) {
                 Setting::set('media_path', $path);
-
                 return;
             }
 
-            $this->error('The path does not exist or not readable. Try again.');
+            $this->error('The path '.$path.' does not exist or not readable. Skipping.');
+            return;
+
+        } else {
```

Comment:
We don't need `else` if early return has been used (that's the whole point of early return here).

Reply:
the `else` is needed in case the user runs `koel:init` in **interactive mode**, like in https://github.com/phanan/koel/pull/886/files/5b9c0342097fd311e2cf7301d5b600eb1fb68ebf#diff-a47e75fc23aa87b0255a42a63499a828R140

Reply:
No, you don't need `else` because you have `return`'ed. Also, please add a new line before `return` statements.

Reply:
I think I finally saw why it didn't required an `else` statement =P, I also tried to add new lines before `returns`, except when they are alone or around single lines.